ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java

```diff
@@ -1,961 +1,961 @@
+	public static LockedAction<Konsultation> moveBehandlungAction, redateAction;
+	public static IAction neuerFallAction;
+
+	public static MenuManager perspectiveMenu, viewMenu;
+	public static IContributionItem viewList;
+	public IWorkbenchWindow mainWindow;
+	public static Action printKontaktEtikette;
+	private static IWorkbenchHelpSystem help;
+	private static Logger logger;
+
+	public GlobalActions(final IWorkbenchWindow window) {
+		if (Hub.mainActions != null) {
+			return;
+		}
+		logger = LoggerFactory.getLogger(this.getClass());
+		mainWindow = window;
+		help = Hub.plugin.getWorkbench().getHelpSystem();
+		exitAction = ActionFactory.QUIT.create(window);
+		exitAction.setText(Messages.GlobalActions_MenuExit); // $NON-NLS-1$
+		newWindowAction = ActionFactory.OPEN_NEW_WINDOW.create(window);
+		newWindowAction.setText(Messages.GlobalActions_NewWindow); // $NON-NLS-1$
+		copyAction = ActionFactory.COPY.create(window);
+		copyAction.setText(Messages.GlobalActions_Copy); // $NON-NLS-1$
+		cutAction = ActionFactory.CUT.create(window);
+		cutAction.setText(Messages.GlobalActions_Cut); // $NON-NLS-1$
+		pasteAction = ActionFactory.PASTE.create(window);
+		pasteAction.setText(Messages.GlobalActions_Paste); // $NON-NLS-1$
+		aboutAction = ActionFactory.ABOUT.create(window);
+		aboutAction.setText(Messages.GlobalActions_MenuAbout); // $NON-NLS-1$
+		// helpAction=ActionFactory.HELP_CONTENTS.create(window);
+		// helpAction.setText(Messages.getString("GlobalActions.HelpIndex"));
+		// //$NON-NLS-1$
+		prefsAction = ActionFactory.PREFERENCES.create(window);
+		prefsAction.setText(Messages.GlobalActions_Preferences); // $NON-NLS-1$
+		savePerspectiveAction = new Action(Messages.GlobalActions_SavePerspective) { // $NON-NLS-1$
+			{
+				setId("savePerspektive"); //$NON-NLS-1$
+				// setActionDefinitionId(Hub.COMMAND_PREFIX+"savePerspektive"); //$NON-NLS-1$
+				setToolTipText(Messages.GlobalActions_SavePerspectiveToolTip); // $NON-NLS-1$
+				setImageDescriptor(Images.IMG_DISK.getImageDescriptor()); // $NON-NLS-1$
+			}
+
+			@Override
+			public void run() {
+				IWorkbenchPage page = mainWindow.getActivePage();
+				if (page != null && page.getPerspective() != null) {
+					page.savePerspectiveAs(page.getPerspective());
+				}
+			}
+		};
+
+		helpAction = new Action(Messages.GlobalActions_ac_handbook) { // $NON-NLS-1$
+			{
+				setImageDescriptor(Images.IMG_BOOK.getImageDescriptor());
+				setToolTipText(Messages.GlobalActions_ac_openhandbook); // $NON-NLS-1$
+
+			}
+
+			@Override
+			public void run() {
+				Desktop desktop = Desktop.getDesktop();
+				String url = CoreHub.globalCfg.get(UiPreferenceConstants.CFG_HANDBOOK,
+						UiPreferenceConstants.DEFAULT_HANDBOOK);
+				if (Desktop.isDesktopSupported()) {
+					try {
+						desktop.browse(new java.net.URI(url));
+					} catch (Exception e) {
+						logger.warn("failed to open default browser :" + e);
+						MessageDialog.openError(mainWindow.getShell(), Messages.GlobalActions_Error,

```

Comment:
Added an error message if user doesn't put a valid URL in Préférences > Iatrix Wiki Hilfe